From da7fd66e1936d8a901472a818db05c631cd78a9d Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Fri, 12 Jun 2015 17:22:12 +0800
Subject: [PATCH] MLK-11100 ARM: dts: imx6ul-14x14-ddr3-arm2-emmc: disable
 qspi when use emmc

eMMC has pin conflict with qspi, so disable qspi when enable emmc on
imx6ul-14x14-ddr3-arm2 board.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-emmc.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-emmc.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-emmc.dts
index 2e35ed6..e1185ce 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-emmc.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2-emmc.dts
@@ -8,6 +8,10 @@
 
 #include "imx6ul-14x14-ddr3-arm2.dts"
 
+&qspi {
+	status = "disable";
+};
+
 &usdhc1 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc1_8bit>;
-- 
1.8.0

